### ISSUE TITLE
Consider running dependency check as an action rather than integrated in the maven build

### ISSUE THREAD

**Issue by netomi** [eclipse-cbi/macos-notarization-service#262]
The dependency check is rather slow as it downloads all cve everytime it is run. 

Consider using an action instead that comes with a pre-build image of cve's so that not all of them have to be downloaded again and again.

**Comment by mbarbero** [comment]
Agreed. Also, with proper dependabot config and dependency graph feeding, dependency-check is less useful. I've configured dependency-check here more to get a feeling about what it could provide.

**Comment by netomi** [comment]
In PR #274 I have move the dependency check to a separate profile that is not enabled by default in the ci builds.

**Comment by mbarbero** [comment]
Should we close this one then? Or do you want to create a workflow that will run with this profile separately?

**Comment by netomi** [comment]
We should add an action to run the dependency check on a regular basis using  a schedule and then we can compare the results with dependabot. I see that as experimenting with existing tools to understand their strengths and weaknesses.
